app/api/prompts.js

```diff
@@ -0,0 +1,17 @@
+const { get } = require('./base')
+const { promptsApi: config } = require('../config/api')
+
+const baseUrl = config.baseUrl
+
+const listPrompts = async (project = 'mcu', model, type) => {
```

Comment:
Should be `getPrompt`

Reply:
this would then clash with getPrompts in `/app/services/prompts.js`